service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationStatementMapper.java

```diff
@@ -161,23 +166,29 @@ private String prepareComment(Observation observation) {
 
                 Optional.ofNullable(coding).ifPresent(code -> {
                     if (code.hasCode() || code.hasDisplay()) {
-                        commentBuilder.append("Interpretation Code: ");
+                        commentBuilder.append(INTERPRETATION_CODE_PREFIX);
                     }
                     if (code.hasCode()) {
                         commentBuilder.append(coding.getCode()).append(StringUtils.SPACE);
                     }
                     if (code.hasDisplay()) {
                         commentBuilder.append(code.getDisplay()).append(StringUtils.SPACE);
                     }
-
-                    if (interpretation.hasText()) {
-                        commentBuilder.append("Interpretation Text: ").append(interpretation.getText()).append(StringUtils.SPACE);
-                    }
                 });
             }
         }
 
-        commentBuilder.append(observation.hasComment() ? observation.getComment() : StringUtils.EMPTY);
+        Optional<String> interpretationText = StringUtils.isNotBlank(observation.getInterpretation().getText())
+            ? Optional.of(observation.getInterpretation().getText())
+            : Optional.empty();
+        String interpretationTextAndComment = Stream.concat(
+            interpretationText.stream(),
+            Optional.ofNullable(observation.getComment()).stream()
+        ).collect(Collectors.joining(StringUtils.SPACE));
```

Comment:
Joining the two parts with a space is what the story requires. I'm wondering, though, if we could replace it with some other character, e.g. `;`? Otherwise, it might be hard to see where one part ends and the other one starts.

Reply:
@dmcavenue1 What do you think about this?
Making this `Interpretation: {{interpretation.text}}; {{comment}}` rather than `Interpretation: {{interpretation.text}} {{comment}}`?

Reply:
I'm not for it - think we already may have too many different delimiters/separators in text so I'm happier with just the space and ; has a source code feel to me

Reply:
Not using any kind of separator we're merging two blocks of tests together without providing any information about where one ends and another one starts. That may be confusing to the person who reads it.

As for the change, I'm fine with it (the space was already present in the code), but just pointing at a potential issue. We can discuss offline and treat it unrelated to this PR.